api-report/odsp-driver.api.md

```diff
@@ -127,6 +128,9 @@ export interface OdspFluidDataStoreLocator extends IOdspUrlParts {
     fileVersion?: string;
 }
 
+// @public
+export function prefetchLatestSnapshot(resolvedUrl: IResolvedUrl, getStorageToken: TokenFetcher<OdspResourceTokenFetchOptions>, persistedCache: IPersistedCache, logger: ITelemetryBaseLogger, hostSnapshotFetchOptions: ISnapshotOptions | undefined): Promise<boolean>;
```

Comment:
@vladsud  is this the right package for this? should this be dynamically loaded? are there versioning implication for this

Reply:
It's fine for now. Basically only driver factory and driver factory input/output times (that are all defined in odsp-driver-definitions) are subject for dynamic loading.
Long term yes, we should separate driver (implementation) into multiple packages, and likely move out error handling code out of odsp-doclib-utils package into its own package.

The tests also validate only that factory is compatible with old versions of loader / runtime (merged this morning).